cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperCluster.java

```diff
@@ -276,9 +276,18 @@ public NetworkPolicy generateNetworkPolicy() {
         labelSelector3.setMatchLabels(expressions3);
         entityOperatorPeer.setPodSelector(labelSelector3);
 
+        NetworkPolicyPeer clusterOperatorPeer = new NetworkPolicyPeer();
+        LabelSelector labelSelector4 = new LabelSelector();
+        Map<String, String> expressions4 = new HashMap<>();
+        expressions4.put(Labels.STRIMZI_KIND_LABEL, "cluster-operator");
+        labelSelector4.setMatchLabels(expressions4);
+        clusterOperatorPeer.setPodSelector(labelSelector4);
+        // This is a hack because we have no guarantee that the CO namespace has some particular labels
```

Comment:
@scholzj what do you mean by this?

Reply:
Ideal, the namespace where the CO is running should have a label and we should use the label in the namespace selector. But since we do not know which label and we cannot even guarantee that it has some label, this is the best we can do - to open it to all COs running in whatever namespace.